client/my-sites/upgrades/checkout-thank-you/domain-mapping-details.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getDomainManagementUrl } from './utils';
+import { getPrimaryDomain, isSubdomain } from 'lib/domains';
+import { isPlan } from 'lib/products-values';
+import PurchaseDetail from './purchase-detail';
+
+export default React.createClass( {
+	getInitialState: function() {
+		return {
+			primaryDomain: null
+		};
+	},
+
+	componentWillMount: function() {
+		getPrimaryDomain( this.props.selectedSite.ID, function( error, data ) {
+			if ( ! error && data ) {
+				this.setState( { primaryDomain: data.domain } );
+			}
+		}.bind( this ) );
```

Comment:
You can get rid of the binding by using an arrow instead since it [binds `this` for free](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions).

![4425067](https://cloud.githubusercontent.com/assets/594356/13226619/ab14dff8-d992-11e5-8fd5-3a8fe23b7515.jpg)